docs/develprocess.md

```diff
@@ -843,4 +894,4 @@ sync;fio --randrepeat=1 --ioengine=libaio --direct=1 --gtod_reduce=1 --name=test
 * Installation of guest tools on new Linux VM
 * Windows VM from an upgraded pool, with older guest drivers not updated
 * Update existing Windows guest drivers
-* Installation of guest drivers on new Windows VM
\ No newline at end of file
+* Installation of guest drivers on new Windows VM
```

Comment:
What's the change here?

Reply:
I guess github's editor added a newline automatically *again*...